pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -56,6 +71,8 @@ export interface PanelOptions {
   usePartialResponse: boolean;
   storeMatches: Store[];
   engine: string;
+  explain: boolean;
+  disableCheckbox: boolean;
```

Comment:
Let's change it to `disableExplainCheckbox` to make it crystal clear we're talking about the "explain" checkbox.